source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,340 @@
+
+1.  Run the `generate:theme` command as shown below to start the process of creating a subtheme.
+
+    ```
+    terminus drupal $SITE.$ENV -- generate:theme
+    ```
+    Drupal Console will ask a series of questions about your theme. For many of them you can use the default value by just hitting **Enter**. Answer these quesions as follows:
+
+    - Theme: Amazing Theme
+    - Base theme: bartik
+    - Enter 'no' to generate theme regions and theme breakpoints
+
+    Type `yes` when asked to confirm generation of the theme. Once you do this, the files for your new theme will be written to the directory `code/web/themes/custom/amazing_theme`. If you gave your theme a different name, replace `amazing_theme` with the appropriate name for your theme.
+
+2.  Use an SFTP client to access the generated theme files in [Pantheon's on-server development mode](https://pantheon.io/docs/sftp/). Find the file `code/web/core/theme/bartik/bartik.info.yml`, and open it in an editor. Copy the regions section starting on line 28, and paste it into the file `code/web/themes/custom/amazing_theme/amazing_theme.info.yml`. Optionally, you might also want to copy the file `code/web/core/theme/bartik/logo.svg` into your theme folder.
+
+3.  In your new theme's directory, create a file named `amazing_theme.libraries.yml`, and put the following contents in it:
+
+    ```
+    global-styling:
+      version: VERSION
+      css:
+        theme:
+          css/main.css: {}
+    ```
+4.  Create a folder named `css` in your theme directory and create a file named `main.css` inside it. To test that your theme is working, add some very obvious styling such as the red border around the content region shown in the example below:
+
+    ```
+    #content {
+      border: 4px solid red;
+    }
+    ```
+
+    Remember to upload your updated files back to the server over SFTP.
+
+5.  Use Terminus with Drupal Console and Drush to active your new theme and rebuild the cache:
+
+    ```
+    terminus drupal $SITE.$ENV theme:install -- set-default amazing_theme
+    ```
+
+    When you view your site in the web browser, it should reflect the change provided by the custom theme.
+
+    ![Modified css](/source/docs/assets/images/pr-workflow/modified-css.png)
+
+6.  Export your configuration, and visit your multidev Site dashboard and commit your changes.
+
+## Behat Tests
+
+As you have already seen, the template project used to create your site included some basic [Behat](http://behat.org/en/latest/) tests that cover some of Drupal's basic capabilities. You can customize these tests or add more to suit your purposes. The file `code/tests/behat-pantheon.yml` controls where tests files are stored. By default, `code/tests/features` and `code/tests/site-features` are the defined search location, but you may add more directories if you would like to organize your tests. Any file with a `.feature` suffix will be executed as part of the standard test run.
```

Comment:
Maybe "Any file with a `.feature` suffix that is stored in one of these locations will be executed as part of the standard test run." would be clearer? Optional.